### DISCUSSION TITLE
Create Audio Component

### DISCUSSION THREAD

**Discussion by levimcg** [indiana-university/rivet-source#408] [Feature and enhancement requests]
Reference existing style guide component:

https://styleguide.iu.edu/pattern-library/chunks/video-audio.html

Remember to account for audio transcripts.

**Comment by zttodd** [comment]
I've been doing some research into this component and it seems that directing developers and users to an accessible third-party library might be the most well-supported route forward for this component and the video component.

At the most basic level, the controls provided from the `controls` attribute in the `audio` player aren't supported the same across browsers (the volume control appears to be missing on the Safari browser, for example). In this case, a JS solution to recreate the HTML5 audio player to be cross-browser supported using the Web Audio API would be needed.

The W3C has a page regarding accessible media players: https://www.w3.org/WAI/media/av/player/. A promising option in that page is the Able Player, which supports both audio and video players. This player provides all sorts of a11y support, including better tabbing and visual indicators, adjustable speed, HOME/END/PAGE UP/PAGE DOWN keys, transcripts, subtitles, chapters, and more.

If we were to go this route, this could be a case where we would provide styles for a "skin" as a Rivet add-on to restyle the players.

This note references the issue for creating a Video Component for Rivet: #286
